.github/workflows/release.yml

```diff
@@ -0,0 +1,70 @@
+name: Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4.1.1
+        with:
+          lfs: true
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4.0.3
+        with:
+          node-version: 16.17
+
+      - name: Enable corepack
+        run: corepack enable yarn
+
+      - name: Bump minor version and save it on environment variable
+        run: |
+          yarn version minor
+          NEW_VERSION=$(node -p "require('./package.json').version")
+          echo "new_version=$NEW_VERSION" >> $GITHUB_ENV
+
+      - name: Bump @lichtblick/suite version
+        run: yarn version minor
+        working-directory: packages/suite
+
+      - name: Install dependencies
+        run: yarn install --immutable
+
+      - name: Build prod files
+        run: |
+          yarn run desktop:build:prod
+
+      - name: Build software in serie
```

Comment:
Suggestion:

Build software for multiple OS
In order to make it more understandable